package.json

```diff
@@ -25,6 +25,8 @@
   "scripts": {
     "prepare": "npm update --dev vl-ui-util && npm run prepare:util",
     "prepare:util": "npm explore vl-ui-util -- npm run util",
+    "performance:generate": "node ./performance/generate.js",
+    "performance:run": "npm run performance:generate && concurrently --kill-others \"npm:demo\" \"node ./performance/run.js\"",
```

Comment:
Waarom de concurrentie kill others?

Reply:
Wanneer een process klaar is, zal het ander ook gestopt worden. Dus als performance/run.js klaar is, zal npm:demo gekilled worden. Anders zal de demo eeuwig blijven open staan.